src/Backend/Modules/FormBuilder/Engine/Model.php

```diff
@@ -549,24 +551,23 @@ public static function update(int $id, array $values): int
         // update item
         $database->update('forms', $values, 'id = ?', $id);
 
-        // build array
-        $extra = [
-            'data' => serialize(
-                [
-                    'language' => BL::getWorkingLanguage(),
-                    'extra_label' => $values['name'],
-                    'id' => $id,
-                    'edit_url' => BackendModel::createUrlForAction('Edit') . '&id=' . $id,
-                ]
-            ),
-        ];
-
-        // update extra
-        $database->update(
-            'modules_extras',
-            $extra,
-            'module = ? AND type = ? AND sequence = ?',
-            ['FormBuilder', 'widget', '400' . $id]
+        // @todo this should be handled in a message handler
+        /** @var ModuleExtraRepository $moduleExtraRepository */
+        $moduleExtraRepository = BackendModel::get(ModuleExtraRepository::class);
+
+        if (!$moduleExtraRepository instanceof ModuleExtraRepository) {
```

Comment:
I would actually just remove this everywhere. The container will already throw an `ServiceNotFoundException`exception if the service wasn't found by default, if you want a null you need to pass `ContainerInterface::NULL_ON_INVALID_REFERENCE` and if somebody overwrites this repository it's their own fault